.github/workflows/integration_tests.yml

```diff
@@ -33,6 +46,7 @@ jobs:
       run: echo "::set-output name=date::$(date +'%Y-%m-%d' -d "last saturday")"
         
     - name: Download Cached GCC Binaries
+      if:  ${{ ${{ matrix.config.toolchain }} == "gcc" }}
```

Comment:
@ParkerJones567 Shouldn't GCC also be required for the LLVM build?

Reply:
Yes, I just pushed a fix for that, everything should be working now, the tests are running right now